qa/tasks/cephfs/test_volumes.py

```diff
@@ -95,6 +95,30 @@ def tearDown(self):
             self._delete_test_volume()
         super(TestVolumes, self).tearDown()
 
+    def test_volume_rm(self):
+        try:
+            self._fs_cmd("volume", "rm", self.volname)
+        except CommandFailedError as ce:
+            if ce.exitstatus != errno.EPERM:
+                raise
+            else:
+                try:
+                    self._fs_cmd("volume", "rm", self.volname, "--yes-i-really-mean-it")
+                except CommandFailedError as ce:
+                    #re-raise EPERM
+                    raise
+
+                #check if it's gone
+                try:
+                    self._fs_cmd("volume", "rm", self.volname, "--yes-i-really-mean-it")
```

Comment:
To check whether volume exists you can do thism https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/filesystem.py#L601